README.md

```diff
@@ -331,6 +335,9 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Jellyfin**](https://github.com/jellyfin/jellyfin-android) <sup>**[[F-Droid](https://f-droid.org/app/org.jellyfin.mobile)]**</sup>
 * [**Twire**](https://github.com/twireapp/Twire) <sup>**[[F-Droid](https://f-droid.org/app/com.perflyst.twire)]**</sup>
 * [**LibreTube**](https://github.com/libre-tube/LibreTube) <sup>**[[F-Droid](https://f-droid.org/app/com.github.libretube)]**</sup>
+* [**VueTube**](https://github.com/VueTubeApp/VueTube)
+* [**SmartTubeNext**](https://github.com/yuliskov/SmartTubeNext)
```

Comment:
It contains quite some proprietary and tracking libraries importunately: https://github.com/yuliskov/SmartTubeNext/issues/471#issuecomment-858083371

The comment is from last year, but the situation seems still the same.